Server-2047 | Simplified flyway & postgres chart migration (#7305)
circleci/circleci-docs: jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc

----

IMPORTANT: If your Postgres instance is not externalized, when upgrading to CircleCI Server v4.0, the Postgres chart is also being upgraded.

[#check-migrator-job-status]
=== 3. Check migrator job status
Check if migrator job status if exists -
[source,shell]
----
kubectl -n <namespace> get job/circle-migrator
----

> If the job does not exist, check the pod's logs `circle-migrator.log` in the `log` directory. An example of the logs can be found below.

Reviewer: ```suggestion
If the job does not exist, check the pod's logs (`circle-migrator.log`) in the `log` directory. An example of the logs can be found below:
```

----

IMPORTANT: If your Postgres instance is not externalized, when upgrading to CircleCI Server v4.0, the Postgres chart is also being upgraded.

[#check-migrator-job-status]
=== 3. Check migrator job status
Check if migrator job status if exists -
[source,shell]
----
kubectl -n <namespace> get job/circle-migrator
----

> If job does not exists then check the pod log `circle-migrator.log` in `log` directory and pod log can be is look like below -

Reviewer 2: ```suggestion
If the job does not exist, check the pod's logs `circle-migrator.log` in the `log` directory. An example of the logs can be found below.
```

> [source,shell]

Reviewer 2: is shell highlighting the best here?

----
Successfully baselined schema with version: 1
Current version of schema "public": 1
DEBUG: Parsing V0002__drop_domain_migrations.sql ...
DEBUG: Found statement at line 1: DROP TABLE IF EXISTS public.domain_migrations
DEBUG: Starting migration of schema "public" to version "0002 - drop domain migrations" ...
Migrating schema "public" to version "0002 - drop domain migrations"
DEBUG: Executing SQL: DROP TABLE IF EXISTS public.domain_migrations
DEBUG: 0 rows affected
DEBUG: Successfully completed migration of schema "public" to version "0002 - drop domain migrations"
DEBUG: Schema History table "public"."schema_version" successfully updated to reflect changes
Successfully applied 1 migration to schema "public", now at version v0002 (execution time 00:00.101s)
DEBUG: Memory usage: 12 of 15M
----

> Else, contact with your circleci point of contact for further instructions.

Reviewer 3: ```suggestion
Else, reach out to your circleci point of contact for further instructions.
```

> Else, reach out to your circleci point of contact for further instructions.

Reviewer 2: ```suggestion
For any other concerns, please reach out to your CircleCI point of contact.
```

Do we actually even need this sentence?
